spec/lib/stripe/credit_card_remover_spec.rb

```diff
@@ -6,42 +6,44 @@
 describe Stripe::CreditCardRemover do
   let(:credit_card) { double('credit_card', gateway_customer_profile_id: 1) }
 
-  context 'Stripe customer exists' do
-    context 'and is not deleted' do
-      it 'deletes the credit card clone and the customer' do
-        customer = double('customer', deleted?: false)
-        allow(Stripe::Customer).to receive(:retrieve).and_return(customer)
+  describe "#remove", :vcr, :stripe_version do
+    context 'Stripe customer exists' do
+      context 'and is not deleted' do
+        it 'deletes the credit card clone and the customer' do
+          customer = double('customer', deleted?: false)
+          allow(Stripe::Customer).to receive(:retrieve).and_return(customer)
+
+          expect_any_instance_of(Stripe::CreditCardCloneDestroyer).to receive(:destroy_clones).with(
+            credit_card
+          )
+          expect(customer).to receive(:delete)
+          Stripe::CreditCardRemover.new(credit_card).call
+        end
+      end
 
-        expect_any_instance_of(Stripe::CreditCardCloneDestroyer).to receive(:destroy_clones).with(
-          credit_card
-        )
-        expect(customer).to receive(:delete)
-        Stripe::CreditCardRemover.new(credit_card).call
+      context 'and is deleted' do
+        it 'deletes the credit card clone' do
+          customer = double('customer', deleted?: true)
```

Comment:
Same as above

Reply:
Hum. I believe I must have screwed up my git history: this should not be here, in fact it is possible to delete the customer within a before block, and then run the test... I'll to replace this stub as well. Moving back to in dev.